test/spacerequest/spacerequest_assertions.go

```diff
@@ -69,6 +70,35 @@ func (a *Assertion) HasStatusTargetClusterURL(targetCluster string) *Assertion {
 	return a
 }
 
+func (a *Assertion) HasNamespaceAccess(namespaceAccess []toolchainv1alpha1.NamespaceAccess) *Assertion {
+	err := a.loadResource()
+	require.NoError(a.t, err)
+	assert.True(a.t, len(a.spaceRequest.Status.NamespaceAccess) > 0)
```

Comment:
this should be
```suggestion
	assert.Len(a.t, a.spaceRequest.Status.NamespaceAccess, len(namespaceAccess))
```
otherwise, if the number of namespaces in the status is bigger (it contains more) than the current test wouldn't detect that.